components/net_traits/lib.rs

```diff
@@ -504,6 +506,14 @@ impl ResourceFetchTiming {
     // TODO currently this is being set with precise time ns when it should be time since
     // time origin (as described in Performance::now)
     pub fn set_attribute(&mut self, attribute: ResourceAttribute) {
+        let should_attribute_always_be_updated = match attribute {
+            ResourceAttribute::FetchStart => true,
+            ResourceAttribute::ResponseEnd => true,
```

Comment:
Looks like we should let StartTime pass through as well, based on https://w3c.github.io/resource-timing/#processing-model.

Reply:
I'm not sure what you mean. StartTime seems to be in PerformanceEntry so it shouldn't be accessible through ResourceFetchTiming, right?

Reply:
Indeed. Nevermind!